ui/apps/platform/src/services/GroupsService.ts

```diff
@@ -1,4 +1,5 @@
 import axios from './instance';
+import { GetDefaultGroupRequest, Group, GroupBatchUpdateRequest } from '../types/group.proto';
```

Comment:
For symmetry, because proto/api/v1/group_service.proto encapsulates `GroupBatchUpdateRequest` and `GetDefaultGroupRequest` is frontend only subset of `GetGroupsRequest` type, can you encapsulate those types in this file and import only `Group` type?

Reply:
Also, you can write `from 'types/group.proto;' without `..` because there is some magic to provide relative paths from `src` folder.